e2e/cypress/integration/enterprise/system_console/channel_moderation_spec.js

```diff
@@ -35,24 +60,925 @@ const enableAllChannelModeratedPermissions = () => {
     });
 };
 
+// # Enable (check) all the permissions in the channel moderation widget through the API
+const enableAllChannelModeratedPermissionsViaAPI = (channelId) => {
+    cy.externalRequest(
+        {
+            user: users.sysadmin,
+            method: 'PUT',
+            path: `channels/${channelId}/moderations/patch`,
+            data:
+                [
+                    {
+                        name: 'create_post',
+                        roles: {
+                            members: true,
+                            guests: true,
+                        }
+                    },
+                    {
+                        name: 'create_reactions',
+                        roles: {
+                            members: true,
+                            guests: true,
+                        }
+                    },
+                    {
+                        name: 'manage_members',
+                        roles: {
+                            members: true,
+                        }
+                    },
+                    {
+                        name: 'use_channel_mentions',
+                        roles: {
+                            members: true,
+                            guests: true,
+                        }
+                    },
+                ],
+        }
+    );
+};
+
+const deleteExistingTeamOverrideSchemes = () => {
+    cy.apiLogin('sysadmin');
+    cy.apiGetSchemes('team').then((res) => {
+        res.body.forEach((scheme) => {
+            cy.apiDeleteScheme(scheme.id);
```

Comment:
I think you need to add this method to the api commands file too

Reply:
True that